test/common/idpay/envVars.js

```diff
@@ -33,8 +34,9 @@ export const idpayDefaultHeaders = Object.assign(
 const ioBaseUrl = getBaseUrl([DEV, UAT, PROD], 'io')
 export function buildIOAuthorizationHeader(fiscalCode) {
     const authToken = loginFullUrl(
-        `${ioBaseUrl}/bpd/pagopa/api/v1/login`,
-        fiscalCode
+        `${ioBaseUrl}/rtd/mock-io/login`,
+        fiscalCode,
+        IDPAY_CONFIG.AUTH_KEYS.APIM_LOGIN_KEY
```

Comment:
this is wrong, or better, this is not what was asked to change observing
https://github.com/pagopa/cstar-integration-test/pull/80/files

You are supposed to call getMockedIoToken instead of loginFullUrl!